### PR TITLE
packages/devel - next batch of bumps

### PR DESCRIPTION
**Pull request by heitbaum**
boost: update to 1.75.0 - single bump
elfutils: update to 0.182 - minor (12 months) 
cmake: update to 3.19.2 - minor bump

### CODE REVIEW

**Comment by heitbaum** [comment]
Dropping automake for the moment. Can see that @MilhouseVH alreadynidentified the incompatibility. https://github.com/heimdal/heimdal/issues/529

File: packages/devel/elfutils/package.mk [hunk 1]
```diff
@@ -26,6 +26,7 @@ PKG_CONFIGURE_OPTS_HOST="utrace_cv_cc_biarch=false \
                          --disable-programs \
                          --disable-nls \
                          --disable-debuginfod \
+			 --disable-libdebuginfod \
```
Comment (lrusak) [inline packages/devel/elfutils/package.mk:29]: indentation

File: packages/devel/elfutils/package.mk [hunk 2]
```diff
@@ -34,6 +35,7 @@ PKG_CONFIGURE_OPTS_TARGET="utrace_cv_cc_biarch=false \
                            ${PKG_PROGRAMS} \
                            --disable-nls \
                            --disable-debuginfod \
+			   --disable-libdebuginfod \
```
Comment (lrusak) [inline packages/devel/elfutils/package.mk:38]: indentation

**Review by lrusak — CHANGES_REQUESTED** [review]
Just some minors, otherwise looks fine

File: packages/devel/elfutils/package.mk (hunk 1 above)
Comment (heitbaum) [reply packages/devel/elfutils/package.mk]: Love them tabs

File: packages/devel/elfutils/package.mk (hunk 2 above)
Comment (heitbaum) [reply packages/devel/elfutils/package.mk]: Love them tabs

File: packages/devel/elfutils/package.mk (hunk 1 above)
Comment (CvH) [reply packages/devel/elfutils/package.mk]: we don't do tabs at all :) and v4l-utils are likely a mistake

```
$ grep -rP '\t' --include="package.mk" ./packages/
./packages/sysutils/v4l-utils/package.mk:	--enable-bpf \
./packages/sysutils/v4l-utils/package.mk:	--enable-static \
./packages/sysutils/v4l-utils/package.mk:	--disable-shared \
./packages/sysutils/v4l-utils/package.mk:	--disable-doxygen-doc"
./packages/sysutils/v4l-utils/package.mk:# *		rc-rc6-mce	rc6_mce.toml
./packages/sysutils/v4l-utils/package.mk:*		rc-rc6-mce	libreelec_multi.toml
./packages/sysutils/v4l-utils/package.mk:meson-ir	rc-empty	libreelec_multi.toml
./packages/tools/rpi-eeprom/package.mk:	_basedir="$(basename "${_dir}")"
```

Comment (heitbaum) [reply packages/devel/elfutils/package.mk]: 👍. Not a fan either.

Comment (heitbaum) [reply packages/devel/elfutils/package.mk]: Fixed the 2 other package.mk files - https://github.com/LibreELEC/LibreELEC.tv/pull/4814 remaining 3 lines of tabs in the v4l-utils/package.mk are supposed to be there.

**Review by lrusak — APPROVED** [review]
LGTM

**Comment by vpeter4** [comment]
Seems elfutils:host depends on a zstd library.
Solution is to use `--without-zstd` for host.

**Pull request merged**